lightning/src/ln/channel.rs

```diff
@@ -3678,6 +3678,29 @@ impl<Signer: Sign> Channel<Signer> {
 		Ok((None, timed_out_htlcs))
 	}
 
+	/// Indicates the funding transaction is no longer confirmed in the main chain. This may
+	/// force-close the channel, but may also indicate a harmless reorganization of a block or two
+	/// before the channel has reached funding_locked and we can just wait for more blocks.
+	pub fn funding_transaction_unconfirmed(&mut self) -> Result<(), msgs::ErrorMessage> {
+		if self.funding_tx_confirmation_height != 0 {
+			// We handle the funding disconnection by calling update_best_block with a height one
+			// below where our funding was connected, implying a reorg back to conf_height - 1.
+			let reorg_height = self.funding_tx_confirmation_height - 1;
+			let best_time = self.update_time_counter;
```

Comment:
Maybe a comment for why this stays the same? Seems like since it's just for a timestamp, it'd still make sense to be incremented..